cms/templates/js/active-video-upload.underscore

```diff
@@ -1,3 +1,8 @@
 <h4 class="video-detail-name"><%- fileName %></h4>
 <progress class="video-detail-progress" value="<%= progress %>"></progress>
-<p class="video-detail-status"><%- gettext(status) %></p>
+<p class="video-detail-status">
+  <%- gettext(
```

Comment:
To clarify you say, " The string to be extracted should always be in the same line with the function name." - so why is this not in the same line?